service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/dynamic/DynamicList.java

```diff
@@ -26,23 +25,6 @@
     @JsonProperty("list_items")
     private List<DynamicListElement> listItems;
 
-    /**
-     * Converts a list of elements to the appropriate structure to then be handled correctly by CCD.
-     *
-     * @param elements elements to convert into options for the dropdown
-     * @param selected a {@link DynamicListElement} that will be the default selected element of the list
-     * @param <T>      a class that implements {@link DynamicElementIndicator#toDynamicElement()}
-     * @return a {@link DynamicList} to be sent to CCD
-     */
-    public static <T extends DynamicElementIndicator> DynamicList toDynamicList(List<T> elements,
-                                                                                DynamicListElement selected) {
-        List<DynamicListElement> items = elements.stream()
-            .map(DynamicElementIndicator::toDynamicElement)
-            .collect(Collectors.toList());
-
-        return DynamicList.builder().listItems(items).value(selected).build();
-    }
```

Comment:
I think after those changes interface DynamicElementIndicator is no longer used please remove.